### PR TITLE
[FSK-116]: List Of Projects Page Implementation

### PR DESCRIPTION
**Pull request by erfan-goodarzi**
## Changes 

- Create `ProjectList` component.
- Create `ProjectTable` component.
- Add component test and storybook.
- Make `ActionButton` reusable.

### CODE REVIEW

File: src/components/FamilyList/FamilyTable/FamilyTableRow.tsx [hunk 1]
```diff
@@ -38,27 +36,12 @@ export const FamilyTableRow = ({
...
-                }
-              >
-                <MenuIcon />
-              </ActionIcon>
-            </Menu.Target>
-          </Menu>
+          <ActionButton
+            MenuButtonId={familyTableMenuButtonId}
+            MenuId={familyTableMenuId}
+            to="/dashboard/families/:id"
```
Comment (AmirabbasJ) [inline src/components/FamilyList/FamilyTable/FamilyTableRow.tsx:42]: we should use AppRoute here

File: src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    it('should close their menu on second click', () => {
+      cy.findAllByTestId(projectTableMenuButtonId).each(el => {
+        const menuBtn = cy.wrap(el);
+        menuBtn.click();
+        menuBtn.click();
+        cy.findByTestId(projectTableMenuId).should('not.exist');
+      });
+    });
+  });
+});
```
Comment (AmirabbasJ) [inline src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx:30]: I guess now that you abstracted Action Buttons we should test them there not in places where it is uesed

**Review by AmirabbasJ — CHANGES_REQUESTED** [review]

File: src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx (hunk 2 above)
Comment (erfan-goodarzi) [reply src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx:30]: U mean in atoms?

Comment (AmirabbasJ) [reply src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx:30]: yeah

Comment (erfan-goodarzi) [reply src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx:30]: I don't know. I'm not sure.

Comment (AmirabbasJ) [reply src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx:30]: what's wrong with doing it?

Comment (ASafaeirad) [reply src/components/ProjectList/ProjectTable/ProjectTable.cy.tsx:30]: Agree.

**Review by ASafaeirad — APPROVED** [review]

**Pull request merged**